### PR TITLE
Add kcc mainnet and testnet to setup_service

### PR DESCRIPTION
**Pull request by mhxw**
### What was wrong? 

kcc mainnet and testnet were missing the setup service.

### CODE REVIEW

**Comment by Uxio0** [comment]
Please add link to https://github.com/safe-global/safe-deployments/ PR with this network and accept the CLA

**Comment by mhxw** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by mhxw** [comment]
> Please add link to https://github.com/safe-global/safe-deployments/ PR with this network and accept the CLA

https://github.com/safe-global/safe-deployments/pull/122

**Comment by Uxio0** [comment]
We will merge this after `safe-deployments` is merged

**Comment by mhxw** [comment]
> We will merge this after [safe-global/safe-deployments#124](https://github.com/safe-global/safe-deployments/issues/124) is merged
safe-deployments is merged

**Comment by mhxw** [comment]
> 

When will the docker image be updated?

> > We will merge this after [safe-global/safe-deployments#124](https://github.com/safe-global/safe-deployments/issues/124) is merged
> > safe-deployments is merged

File: safe_transaction_service/history/management/commands/setup_service.py
Comment (Uxio0) [inline safe_transaction_service/history/management/commands/setup_service.py:479]: This address is not right

**Review by Uxio0 — CHANGES_REQUESTED** [review]

**Comment by Uxio0** [comment]
Could you rebase master so I can merge?

**Comment by mhxw** [comment]
> > We will merge this after [safe-global/safe-deployments#124](https://github.com/safe-global/safe-deployments/issues/124) is merged
> > Thanks, safe-deployments is merged，when will the docker image be updated? I found the dockerfile to make the image in the cgw and cfg repositories, but in the current safe-transaction-service repository, I don't find the image to make the service?

Is the latest tab of this mirror automatically pushing the new mirror after merging the code?

**Comment by mhxw** [comment]
> 

https://github.com/safe-global/safe-transaction-service/pull/1108/commits/9b6f2c5d33adabd3710f85c20a1a73425902159c

**Comment by mhxw** [comment]
> Could you rebase master so I can merge?



> Could you rebase master so I can merge?

https://github.com/safe-global/safe-transaction-service/pull/1108/commits/9b6f2c5d33adabd3710f85c20a1a73425902159c

**Comment by Uxio0** [comment]
> > Could you rebase master so I can merge?
> 
> Is this what you mean? [9b6f2c5](https://github.com/safe-global/safe-transaction-service/commit/9b6f2c5d33adabd3710f85c20a1a73425902159c)

No, I mean doing `git rebase master`, as there are conflicts and I cannot merge your PR 😉

**Comment by mhxw** [comment]
> No, I mean doing `git rebase master`, as there are conflicts and I cannot merge your PR 😉

OK, please merge 😉

**Pull request merged**
